### ISSUE TITLE
config set does not support new centos iso-url option

### ISSUE THREAD

**Issue by jimmidyson** [minishift/minishift#1070]
```bash
$ minishift config set iso-url centos
[centos url is not valid: parse centos: invalid URI for request]
```

centos needs to be added as a valid value to the validator.

Great release btw!

**Comment by hferentschik** [comment]
> centos needs to be added as a valid value to the validator.

Good catch, thank you.

> Great release btw!

Thanks :-) On On

**Comment by hferentschik** [comment]
Merged via pull request #1118
